client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/taxi_gateway/TaxiGateway.kt

```diff
@@ -0,0 +1,19 @@
+package org.thechance.common.data.remote.gateway.taxi_gateway
+
+import org.thechance.common.domain.entity.AddTaxi
+import org.thechance.common.domain.entity.DataWrapper
+import org.thechance.common.domain.entity.Taxi
+
+class TaxiGateway:ITaxiGateway {
+    override suspend fun getTaxis(): DataWrapper<Taxi> {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun createTaxi(taxi: AddTaxi): Taxi {
```

Comment:
AddTaxi  data class name changed into NewTaxiInfo so taxi should be of NewTaxiInfo type and not AddTaxi

Reply:
resolved 👍